src/altinn-app-frontend/webpack.common.js

```diff
@@ -45,6 +45,17 @@ module.exports = {
           },
         ],
       },
+      {
+        test: /\.png$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name:'assets/[name].[ext]'
+            },
+          },
+        ],
+      }
```

Comment:
Is the custom name required? If not this is all that is needed:

```suggestion
     {
       test: /\.png/,
       type: 'asset/resource'
     }
```

If a custom name is needed as well, we need to make some other changes as well. Let me know if thats the case 🙂